src/current/_includes/v23.1/ui/statements-filter.md

```diff
@@ -8,16 +8,26 @@
 
 ## Statement Fingerprints results
 
-The statement fingerprints returned are determined by the selected **Search Criteria**:
+The statement fingerprints returned are determined by the selected **Search Criteria**.
 
-1. By default, the **Top** `100` statement fingerprints **By** `% of All Runtime` are returned.
-  - To change the number returned, select `25`, `50`, or `500` from the dropdown.
-  - To change the column filter, select `CPU Time`, `Contention Time`, `Execution Count`, `P99 Latency`, or `Statement Time` from the dropdown.
+### Search Criteria ###
+
+By default, the **Top** `100` statement fingerprints **By** `% of All Runtime` for the `Past Hour` are returned.
+
+1. To change the number of results returned, select `25`, `50`, `100`, or `500` from the **Top** dropdown. To return a larger number, select `More` and choose an option: `1000`, `5000`, `10000`.
+1. To change the sort column, from the **By** dropdown, select a commonly sorted column: `% of All Runtime`, `CPU Time`, `Contention Time`, `Execution Count`, `P99 Latency`, `Statement Time`. To sort by other columns, select `More` from the dropdown and choose an option: `Last Execution Time`, `Max Latency`,`Max Memory`, `Min Latency`, `Network`, `P50 Latency`, `P90 Latency`, `Retries`, `Rows Processed`.
```

Comment:
I thought we had a "None" option as well. To avoid customers having to click through each column to find their 'interesting' query. @maryliag can you confirm?

Reply:
We don't have that yet.

Reply:
We don't have a tracking issue for it